static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,113 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
+to enable reusability of any <abbr>data artifacts</abbr> (raw data, intermediate
+results, models, etc) between different projects. For example, project A may use
+a data file to begin its data [pipeline](/doc/command-reference/pipeline), but
+project B also requires this same file; Instead of
+[adding it](/doc/command-reference/add#example-single-file) it to both projects,
+B can simply import it from A.
+
+Taking this idea to a useful extreme, we could create a <abbr>project</abbr>
+that is exclusively dedicated to
+[tracking and versioning](/doc/use-cases/data-and-model-files-versioning)
+datasets (or any kind of large files) – by mainly using `dvc add` to build it.
+Such a project would not have [stages](/doc/command-reference/run), but its data
+files may be updated manually as they evolve. Other projects can then share
+these artifacts by downloading (`dvc get`) or importing (`dvc import`) them for
+use in different data processes – and these don't even have to be _DVC
+projects_, as `dvc get` works anywhere in your system.
+
+The advantages of using a data registry are:
+
+- Tracked data is stored in a **centralized** remote location, with the ability
+  to create distributed copies on other remotes.
+- Several projects can **share** the same files, guaranteeing that everyone has
+  access to the same data versions. See
+  [Share Data and Model Files](/doc/use-cases/share-data-and-model-files) for
+  more information.
+- Projects that import data from the registry don't need to push these large
+  files to their own [remotes](/doc/command-reference/remote), **saving space**
+  on storage – they may not even need a remote at all, using only their local
+  <abbr>cache</abbr>.
+- DVC data registries can handle multiple versions of data and ML modes with a
+  familiar CLI. See
+  [Data and Model Files Versioning](/doc/use-cases/data-and-model-files-versioning)
+  for more information.
+- DVC data registries are versioned with Git, so you can always track the
+  history of the project the same as you manage your source code repository.
+
+## Example
```

Comment:
Before we jump into example, can we add an overview? How does the workflow looks like from a high level perspective?

Reply:
What workflow?

Reply:
I mean as simple as - You come up with a Git repo that you make a registry. You put your datasets or models there with `dvc add`, `dvc run`, whatnot. Then you use `dvc import` in your other project to use a specific version of a dataset from that Github repo. This is similar to how you would use requirements Python file to specify a dependency on code package but for data. There are other commands like `dvc update` to get a new version of that data artifact.

This way you have a Git repo that has all the metadata and history for the data. You can see how updated what, you can use PRs to update data the same way you do with code. You don't need to come up with some adhoc conventions to place different versions of you data on S3. 

The point is that we jump into pretty a details almost reproducible real-life example w/o giving users a sense to how is it organized. A lot of them can pick the concept from a few paragraphs. I would also say - this is the split between use cases and tutorials. We can for example forward them to a specific tutorial to show exact commands in a specific situation.

Reply:
Any thoughts about this? Should we do a call and discuss?

Reply:
Sorry, didn't have time to address this part yet. But sure any time we can coincide online.

Reply:
Still working on this... I've revised the intros of the other use cases for now. Will report back ASAP.

Reply:
OK I've pushed a new intro trying to incorporate your suggestion and existing info. It may be too long or a little repetitive right now, especially in the benefits bullets section.